.github/automation/aarch64/build.sh

```diff
@@ -33,17 +33,32 @@ ONEDNN_TEST_SET=${ONEDNN_TEST_SET:-"SMOKE"}
 ONEDNN_BUILD_GRAPH=${ONEDNN_BUILD_GRAPH:-"ON"}
 
 if [[ "$ONEDNN_ACTION" == "configure" ]]; then
-    set -x
-    cmake \
-        -Bbuild -S. \
-        -DDNNL_AARCH64_USE_ACL=ON \
-        -DONEDNN_BUILD_GRAPH=$ONEDNN_BUILD_GRAPH \
-        -DDNNL_CPU_RUNTIME=$ONEDNN_THREADING \
-        -DONEDNN_WERROR=ON \
-        -DDNNL_BUILD_FOR_CI=ON \
-        -DONEDNN_TEST_SET=$ONEDNN_TEST_SET \
-        -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE
-    set +x
+    if [[ "$GITHUB_JOB" == "pr-clang-tidy" ]]; then
```

Comment:
(nit) This looks fragile. Can we handle this via script parameters instead?

Reply:
Don't mind changing it, but what do you mean by script parameters in this context? I modelled the current method after x64:

https://github.com/uxlfoundation/oneDNN/blob/491235ed4b75ec31e7d6d7f0045ad2ed52e30bf8/.github/automation/x64/build_linters.sh#L9-L23